### ISSUE TITLE
This is relevant for EVERYONE installing Canarytokens after 29.10.22: Crash on startup

### ISSUE THREAD

**Issue by gearanach** [thinkst/canarytokens-docker#121]
Hello,

since this morning I am not able to build the canarytokens docker containers for switchboard and frontend anymore. I tried:

1. Clone canarytokens-docker repository again
2. Copy .env-Files
3. `docker stop $(docker ps -q)`
4. `docker system prune -a`
5. `docker-compose -f docker-compose.yml up`

Here is the error message:
```bash
switchboard    | Unhandled Error
switchboard    | Traceback (most recent call last):
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 674, in run
switchboard    |     runApp(config)
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/scripts/twistd.py", line 25, in runApp
switchboard    |     runner.run()
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 381, in run
switchboard    |     self.application = self.createOrGetApplication()
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 453, in createOrGetApplication
switchboard    |     application = getApplication(self.config, passphrase)
switchboard    | --- <exception caught here> ---
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 464, in getApplication
switchboard    |     application = service.loadApplication(filename, style, passphrase)
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/application/service.py", line 416, in loadApplication
switchboard    |     application = sob.loadValueFromFile(filename, 'application')
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/persisted/sob.py", line 177, in loadValueFromFile
switchboard    |     eval(codeObj, d, d)
switchboard    |   File "switchboard.tac", line 8, in <module>
switchboard    |     from twisted.application import service, internet
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/twisted/application/internet.py", line 54, in <module>
switchboard    |     from automat import MethodicalMachine
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/automat/__init__.py", line 2, in <module>
switchboard    |     from ._methodical import MethodicalMachine
switchboard    |   File "/usr/local/lib/python2.7/dist-packages/automat/_methodical.py", line 7, in <module>
switchboard    |     from inspect import getfullargspec as getArgsSpec
switchboard    | exceptions.ImportError: cannot import name getfullargspec
switchboard    |
switchboard    |
switchboard    | Failed to load application: cannot import name getfullargspec
switchboard    |
frontend exited with code 1
switchboard exited with code 1
```
The error is identical for both frontend and switchboard.

Any ideas?

Kind Regards
Gearanach

**Comment by gearanach** [comment]
**This is a bug caused by an update from 29.10.2022 to a dependency for one of the packages listed in requirements.txt.**

Fix: Add `Automat==20.2.0` to the requirements.txt.

**Comment by jayjb** [comment]
Hi @gearanach,

Thanks for reporting this. We are looking into it now
